data-serving/reusable-data-service/data_service/model/field.py

```diff
@@ -60,14 +60,21 @@ def from_dict(cls, dictionary):
     def python_type(self) -> type:
         return self.model_type(self.type)
 
-    def dataclasses_tuple(self) -> (str, type, dataclasses.Field):
+    def dataclasses_tuples(self):
```

Comment:
`-> (str, type, dataclasses.Field)`?

Reply:
should be `-> List[(str, type, dataclasses.Field)]` but I couldn't get the type checker in Python 3.10 to accept that. Maybe a bug in the type checker? Anyway it wouldn't load the module so I removed the type hint.

Reply:
list[tuple[str, type, dataclasses.Field]] should work (from typing import List, Tuple not required anymore)